docs/dev/table/tableApi.md

```diff
@@ -2184,6 +2184,17 @@ NUMERIC.atan()
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight java %}
+NUMERIC.atan2(NUMERIC)
```

Comment:
given that both parameters are equally important, we might want to change the syntax to `atan2(Numeric, Numeric)`. IMO, that would be more intuitive. What do you think?

Reply:
Agree I changed the syntax to `atan2(Numeric, Numeric)`